docs/_static/nuts-registry.yaml

```diff
@@ -314,6 +350,36 @@ paths:
                       $ref: '#/components/schemas/Event'
 components:
   schemas:
+    CAListWithChain:
+      required:
+        - chain
+        - CAList
+      properties:
+        chain:
+          type: array
+          description: list of certificates, roots first then intermediates, shared amongst all CA's. PEM encoded.
+          items:
+            type: string
+        CAList:
+          type: array
+          description: list of current active (or will be active) vendor CA's. PEM encoded
+          items:
+            type: string
+    CertificateListWithChain:
```

Comment:
This data type effectively the same as `CertificateListWithChain`?

Reply:
has been removed